app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  has_many :likes
+
+  def update_post_user_counter
+    user.increment!(:posts_counter)
+  end
```

Comment:
- [ ] This same thing here, let's use author instead of user.

```ruby
  def update_post_user_counter
    author.increment!(:posts_counter)
  end
```